woocommerce/woocommerce-android
# Add new profiler step to store creation: Store features

Closes: #9603


### Description

Adds new profiler step to store creation: Store features

The new step is currently inaccessible from the store creation flow, so it is safe to merge it. Also, when reviewing the design matches the Figma design, keep in mind the progress bar shown at the top is not implemented yet.

### Testing instructions

- Apply the following patch
```diff
Subject: [PATCH] Add new store challenges fragment to navigation
---
Index: WooCommerce/src/main/kotlin/com/woocommerce/android/ui/login/storecreation/countrypicker/CountryPickerFragment.kt
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/WooCommerce/src/main/kotlin/com/woocommerce/android/ui/login/storecreation/countrypicker/CountryPickerFragment.kt b/WooCommerce/src/main/kotlin/com/woocommerce/android/ui/login/storecreation/countrypicker/CountryPickerFragment.kt
--- a/WooCommerce/src/main/kotlin/com/woocommerce/android/ui/login/storecreation/countrypicker/CountryPickerFragment.kt	(revision 785d102cc447a89df7f53b455e5158004be3ed31)
+++ b/WooCommerce/src/main/kotlin/com/woocommerce/android/ui/login/storecreation/countrypicker/CountryPickerFragment.kt	(date 1692108464578)
@@ -56,8 +56,11 @@
     }
 
     private fun navigateToInstallationStep() {
+//        findNavController().navigateSafely(
+//            CountryPickerFragmentDirections.actionCountryPickerFragmentToSummaryFragment()
+//        )
         findNavController().navigateSafely(
-            CountryPickerFragmentDirections.actionCountryPickerFragmentToSummaryFragment()
+            CountryPickerFragmentDirections.actionCountryPickerFragmentToStoreProfilerChallengesFragment()
         )
     }
 

```
- Trigger store creation flow and after store profiler challenges step, you should land in the new step. 
- Check the UI matches Figma design: 8sNT07kW3VEdnor4baI6ft-fi-400%3A64326

### Images/gif



https://github.com/woocommerce/woocommerce-android/assets/2663464/60ec6384-baf0-420d-ba26-7cd024dd3212
